Relativity.DataExchange.Client.SDK.md

```diff
@@ -15,3 +15,18 @@ Initial Version
 Lowest Version | Highest Version
 --- | ---
 9.7 | 12.1
+
```

Comment:
@krzysztof-geleta can you put the latest version on top?  We want the latest releases on top so users can immediately see them.

Reply:
done